packages/store/src/__tests__/workspace.unit.spec.ts

```diff
@@ -88,6 +89,7 @@ describe.concurrent('basic', () => {
         versions: {},
       },
       [spaceId]: {},
+      [tagsId]: {},
```

Comment:
Here is not using the page-level tags, but the workspace level. Is that expected?

Reply:
Yep

Reply:
Because tagsId is `tags:page0` which is correct, I think. put the tag map under YDoc is not a big deal

Reply:
The top-level entries will be SubDocs, so putting tags into the PageBlock under `space:page0` should make more sense here.

Reply:
Make sense, but I will do this in the end as this not a big deal for now